pyproject.toml

```diff
@@ -57,7 +64,10 @@ jsonpath-ng = { version = "^1.5.3", optional = true }
 openpyxl = { version = "^3.0.0", optional = true }
 progressbar2 = { version = "^4.0.0", optional = true }
 deltalake = { version = ">=0.6.4,<0.11.0", optional = true }
-geopandas = {version = "^0.13.2", optional = true}
+geopandas = [
+    { version = "^0.13.2", markers = "python_version < \"3.12\"", optional = true },
+    { version = "^0.14.1", markers = "python_version >= \"3.12\"", optional = true }
```

Comment:
Do we need to do this? Can we just bump to 0.14.1+?

Reply:
Geopandas drops support for Python 3.8, so it's the same problem as with numpy.